### ISSUE TITLE
508-Description text for input fields(checkbox/select) is not exposed to screen readers.

### ISSUE THREAD

**Issue by knamburigsa** [GSA/sam-design-system#1179]
**Components Involved**
Please list the common component(s) involved that you're experiencing an issue with
checkbox, select

**Expected Behavior**
How you expect the listed component(s) to function
Description text added to input fields should be recognized by screen readers.

**Actual Behavior**
How the listed component(s) actually function
Screen readers do not detect description text for input fields.

**Your Environment**
Angular Version: X.Y.Z
@gsa-sam packages version: X.Y.Z (Please list individually if all packages are not same version)
![image](https://user-images.githubusercontent.com/92172714/236827731-082069f5-0241-413d-b0a2-99afc179b23e.png)

**Ticket Number**
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAEMOD-12993

**Additional context**
Screenshots / recordings, minimal reproduction on stackblitz, steps to reproduce, code snippets, etc.

**Comment by yerramshilpa** [comment]
The issue was resolved in sam-formly@14.0.3 version
